### ISSUE TITLE
Problem: Almon can't publish articles

### ISSUE THREAD

**Issue by zolotokrylin** [holdex/holdex-venture-studio#391]
_Originally posted by @zolotokrylin in https://github.com/holdex/holdex-venture-studio/issues/379#issuecomment-1592804256_

**Comment by georgeciubotaru** [comment]
@almon-s Please provide your `email` and  holdex.io user `slug`

**Comment by almon-s** [comment]
@georgeciubotaru its almon.subba@gmail.com 

> holdex.io user `slug`

I don't think i've got this yet?

**Comment by georgeciubotaru** [comment]
@almon-s Try to login on `https://old.holdex.io` then
![image](https://github.com/holdex/holdex-venture-studio/assets/8217518/fa2a9184-0a41-4565-9d30-b81abdeaf6c8)
and place here the URL of the page

**Comment by almon-s** [comment]
@georgeciubotaru https://holdex.io/almon-subba is this it?

**Comment by georgeciubotaru** [comment]
@almon-s Please give it a try,

You have access to the `almon-subba` user for the production environment

**Comment by almon-s** [comment]
@zolotokrylin what else here? How can i try/test this out? Thanks

**Comment by zolotokrylin** [comment]
@almon-s 
![image](https://github.com/holdex/holdex-venture-studio/assets/1125014/c79f27ed-9307-4b5a-812b-a8266d9bb8c7)
![image](https://github.com/holdex/holdex-venture-studio/assets/1125014/d516e3c3-c262-4a4d-9b50-9ac530769b40)

there is a switcher PRODUCTION/STAGE.
Don't publish any new documents on production yet; but if you fix an existing doc, you are welcome to publish it

**Comment by almon-s** [comment]
@zolotokrylin was trying to publish a minor fix for our existing [services doc](https://docs.google.com/document/d/1dbip6OAfcoPis6fxjTMK_eG_6uyeND9TVZPAQUdf828/edit), I don't seem to have `holdex thread export` extension visible. Do I need to have editing permissions in order to publish?

**Comment by zolotokrylin** [comment]
@almon-s you need to install the extension first. Did you install it? https://workspace.google.com/marketplace/app/holdex_thread_export/866265386978

**Comment by almon-s** [comment]
@zolotokrylin yes I tried looking it up but didn't find anything and with this link I get error

<img width="1680" alt="Screenshot 2023-07-03 at 4 21 21 PM" src="https://github.com/holdex/holdex-venture-studio/assets/112711034/cfca9fc8-2f16-4913-be67-731475bc892b">

**Comment by zolotokrylin** [comment]
Oh.. I know what's the problem. You are not part of our Google Organisation.

**Comment by almon-s** [comment]
>  I know what's the problem. You are not part of our Google Organisation.
We will need to wait for George to come back so we can resolve it. He is back in 2 weeks.

@georgeciubotaru FYI thanks

**Comment by almon-s** [comment]
Hi @georgeciubotaru can you add me to Google Organisation so that i can download the extension to be able to publish articles and resolve this issue, thank you!

**Comment by georgeciubotaru** [comment]
@almon-s I haven't seen this notification. Sorry about that, will do it ASAP.

**Comment by georgeciubotaru** [comment]
@almon-s Try to access this link: https://workspace.google.com/marketplace/app/holdex_thread_export/866265386978

@zolotokrylin The `extension` is now `Unlisted` and can be accessed in Google Marketplace by anyone with the Direct URL, but can only be installed by `Individual + Admin`, meaning the Almon will request approve and Vadim will accept.

Note: Haven't tried but I think that's the way.

**Comment by georgeciubotaru** [comment]
@zolotokrylin No possible to change `App visibility`, only to create a new app with public visibility

Solutions:
1.  Almon will use a demoX@holdex.io account for his profile
2. We creating an email `almon@holdex.io` or something like that

**Comment by zolotokrylin** [comment]
Another solution we are going to use now is:
1. @almon-s proposes changes to the doc, and 
2. someone (Mark, Vadim, George) publishes the doc after being notified by Almon

**Comment by almon-s** [comment]
@georgeciubotaru how long would this process take? if not long let's do it. I don't want to keep waiting to get articles published and currently I'm blocked (need to publish JetBrains article asap) and don't want to chase Mark or you who are already busy 

> @zolotokrylin No possible to change `App visibility`, only to create a new app with public visibility
> 
> Solutions:
> 
> 1. Almon will use a [demoX@holdex.io](mailto:demoX@holdex.io) account for his profile
> 2. We creating an email `almon@holdex.io` or something like that
